master/reference/node/configuration.md

```diff
@@ -7,41 +7,41 @@ The `calico/node` container is primarily configured through environment variable
+| WAIT_FOR_DATASTORE | Wait for connection to datastore before starting. If a successful connection is not made, node will shutdown. [Default: `false`] | boolean |
+| CALICO_LIBNETWORK_CREATE_PROFILES | Enables creating a Calico profile resource for each Docker network.  When disabled, no profiles will be processed even if manually created. [Default: `true`] | boolean |
+| CALICO_LIBNETWORK_LABEL_ENDPOINTS | Enables copying a subset of the Docker container labels for use as Calico labels on workloadendpoints. [Default: `false`] | boolean |
+| CALICO_LIBNETWORK_ENABLED | Enables running the docker-libnetwork plugin directly in the calico/node container. [Default: `true`] | boolean |
+| CALICO_LIBNETWORK_IFPREFIX | Interface prefix to use for the network interface within the Docker containers that have been networked by the Calico driver. [Default: `cali`] | string |
+| CALICO_NETWORKING_BACKEND | Describes which BGP networking backend to use [Default: `bird`] | gobgp, bird, none |
+| CALICO_IPV4POOL_CIDR | The IPv4 Pool to create if none exists at start up. It is invalid to define this variable and NO_DEFAULT_POOLS. [Default: `192.168.0.0/16`] | IPv4 CIDR |
+| CALICO_IPV6POOL_CIDR | The IPv6 Pool to create if none exists at start up. It is invalid to define this variable and NO_DEFAULT_POOLS. [Default: `fd80:24e2:f998:72d6::/64`] | IPv6 CIDR |
+| CALICO_IPV4POOL_IPIP | IPIP Mode to use for the IPv4 POOL created at start up. [Default: `off`] | off, always, cross-subnet |
+| CALICO_IPV4POOL_NAT_OUTGOING | Controls NAT Outgoing for the IPv4 Pool created at start up. [Default: `true`] | boolean |
+| CALICO_IPV6POOL_NAT_OUTGOING | Controls NAT Outgoing for the IPv6 Pool created at start up. [Default: `false`] | boolean |
+| CALICO_STARTUP_LOGLEVEL      | The log severity above which startup calico/node logs are sent to the stdout. [Default: `Error`] | DEBUG, INFO, WARNING, ERROR, CRITICAL, or NONE (case-insensitive) |
+| CLUSTER_TYPE | Contains comma delimited list of indicators about this cluster.  Expected to contain information like hosted, services, kubeadm, tectonic, or datastore type. If using Kubernetes API as the datastore then KDD will automatically be added to this list. | k8s, policy, KDD, canal, mesos, docker, rkt, acs, acse, dcos, kubeadm, kubespray, kops, tectonic |
+| ETCD_ENDPOINTS    | A comma separated list of etcd endpoints [Default: `http://127.0.0.1:2379`] (optional) | string |
+| ETCD_KEY_FILE     | Path to the etcd key file, e.g. `/etc/calico/key.pem` (optional)        | string |
+| ETCD_CERT_FILE    | Path to the etcd client cert, e.g. `/etc/calico/cert.pem` (optional)    | string |
+| ETCD_CA_CERT_FILE | Path to the etcd CA file, e.g. `/etc/calico/ca.pem` (optional)          | string |
+| KUBECONFIG | When using the kubernetes datastore, the location of a kubeconfig file to use. | string |
+| K8S_API_ENDPOINT | Location of the Kubernetes API.  Not required if using kubeconfig.       | string |
```

Comment:
I didn't intend for lining that column up in the last few rows to be a thing - it was just a little OCD on my part. Since the update is approved, I'll just go ahead and commit (next time I'm editing this file, I'll undo the change).